django_project/core/settings/project.py

```diff
@@ -82,3 +83,12 @@
     '0.0.0.0',
     'changelog.kartoza.com',
 ]
+
+try:
+    from .secret import STRIPE_SECRET_KEY
+    from .secret import STRIPE_PUBLISHABLE_KEY
+except ImportError:
+    STRIPE_SECRET_KEY = None
+    STRIPE_PUBLISHABLE_KEY = None
+
+CREDIT_PRICE = 20  # Price of each credit
```

Comment:
We already have this as a model property for the project:

![screen shot 2018-05-17 at 11 44 29](https://user-images.githubusercontent.com/178003/40170024-c00cd0c0-59c7-11e8-9afa-1669b81e5d9d.png)

Reply:
ok tim, didn't see that